dev/bots/test.sh

```diff
@@ -44,3 +44,12 @@ fi
 # generate and analyze our large sample app
 dart dev/tools/mega_gallery.dart
 (cd dev/benchmarks/mega_gallery; flutter analyze --watch --benchmark)
+
+if [ -n $COVERAGE_FLAG ]; then
+  GSUTIL=$HOME/google-cloud-sdk/bin/gsutil
+  GCLOUD=$HOME/google-cloud-sdk/bin/gcloud
+
+  $GCLOUD auth activate-service-account --key-file ../gcloud_key_file.json
+  STORAGE_URL=gs://mojo_infra/flutter/coverage/lcov.info
```

Comment:
The security setup is differnet on them.  We don't use mojo_infra currently.  I need to figure out which one I should get the key for.